### DISCUSSION TITLE
New release for v0.3.0

### DISCUSSION THREAD

**Discussion by bcho** [uber-go/mock#73] [General]
Hey, my team is waiting for some fixes in the master branch, I am wondering is that possible to create a new release?

**Comment by sywhang** [comment]
Hey @bcho , sorry about that. We will tag a new release this week.

**Reply by marten-seemann** [reply]
We've been waiting for a new release as well. Even better if https://github.com/uber-go/mock/issues/80 and https://github.com/uber-go/mock/issues/70 could be resolved before the release :).

**Comment by bcho** [comment]
I saw the new release, thanks!
